sky/clouds/gcp.py

```diff
@@ -681,7 +684,10 @@ def _get_feasible_launchable_resources(
             host_vm_type = GCP.get_default_instance_type(
                 cpus=resources.cpus,
                 memory=resources.memory,
-                disk_tier=resources.disk_tier)
+                disk_tier=resources.disk_tier,
+                region=resources.region,
+                zone=resources.zone)
+            logger.info(f'host_vm_type: {host_vm_type}')
```

Comment:
Shall we remove this log or make this more meaningful?

Reply:
Ah right, I forgot to remove this. Thanks for catching this!